yarpl/include/yarpl/observable/Observer.h

```
@@ -21,6 +21,7 @@ class Observer : public virtual Refcounted {
     DCHECK(subscription);
 
     if (subscription_) {
+      DLOG(ERROR) << "attempt to double subscribe";
       subscription->cancel();
```

Do we want to fail loudly/throw an exception here?